NewHorizons/External/Modules/BaseModule.cs

```diff
@@ -71,6 +71,14 @@ public class BaseModule
         /// </summary>
         [DefaultValue(0)] public int gravityVolumePriority = 0;
 
+        /// <summary>
+        /// Apply physics to this planet when you bump into it. Will have a spherical collider the size of surfaceSize. 
+        /// For custom colliders they have to all be convex and you can leave surface size as 0.
```

Comment:
should the convex thing be documented in detailbuilder guy?

alternatively, should this just not be mentioned here since the guys are all made convex most of the time?